File: Modul 2/C/PRAK205.c
#include <stdio.h>
#include <math.h>

int main()
{
    float a, b, c, Keliling, Luas;
    //a sebagai tinggi, b sebagai sisi miring, dan c sebagai alas
    printf("Input\n");
    scanf("%f" "%f", &a, &b);
    printf("\nOutput\n");
    c =  sqrt((b*b)-(a*a));
    Keliling = a+b+c;
    Luas = c*a/2;
    printf("Alas = %.0f cm\n", c);
    printf("Tinggi = %.0f cm\n", a);
    printf("Keliling = %.0f cm\n", Keliling);
    printf("Luas = %.0f cm^2", Luas);
    return 0;
}

File: Modul 2/Python/PRAK205.py
print("Input")
angka = input().split()
A, B =  map(int, angka)
Tinggi = A
Alas = ((B*B) - (A*A))**0.5 
Keliling = A + Alas + B
Luas = Alas*A/2
print("\nOutput")
print(f"Alas = {Alas:.0f}")
print(f"Tinggi = {Tinggi:.0f}")
print(f"Keliling = {Keliling:.0f}")
print(f"Luas = {Luas:.0f} cm^2")